sandbox/basic.py

```diff
@@ -141,8 +141,12 @@ def key_x(self):
         self.panic(self.tree)
 
 
-css_file = Path(__file__).parent / "basic.css"
-app = BasicApp(css_file=str(css_file), watch_css=True, log="textual.log")
+sandbox_folder = Path(__file__).parent
```

Comment:
I'm not keen on using `__file__` because I don't want it to emerge as a standard. There's no `__file__` when running interactively.

In the case of the sandbox, do we need it at all? I run the scripts with sandbox as the CWD.

Reply:
True! I'll remove the use of `Path` here (but I'll keep it in the e2e copy :-)